### PR TITLE
added more instructions

### PR DESCRIPTION
**Pull request by nlharris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ If the above command fails or if it reports a version number significantly diffe
...
 
+Clone this repo:
+```
+git clone https://github.com/monarch-initiative/monarch-ui
+```
+Go into the directory that has your newly cloned copy of this repo:
+```
+cd monarch-ui
+```
+Then you can run the installation setup:
```
Comment (kshefchek) [inline README.md]: Should we add a note here about node and npm versions and potentially link to [installing-nodejs](#installing-nodejs)?

Comment (nlharris) [reply README.md]: sure, do you want to add something? i did the npm setup a while ago and can't remember what exactly i did.

Comment (DoctorBud) [reply README.md]: These instructions are an improvement, but not quite correct. We want to tell people to FORK that repo, and then to CLONE their fork. This has been excruciating detailed in Legacy here: https://github.com/monarch-initiative/monarch-app/blob/master/CONTRIBUTING.md and much of that document should be repurposed for monarch-ui (I honestly thought I had done this a while ago).

**Comment by DoctorBud** [comment]
Closing this PR because it is subsumed by the 0.0.18 PR that I will be submitting soon. 0.0.18 is primarily a documentation PR and it includes the amendments you suggested @nlharris

**Pull request closed without merging**